_data/navbar.yml

```diff
@@ -0,0 +1,11 @@
+# note: key must match page title key as used in _i18n/en.yml
+- key: about
```

Comment:
I think this should be `titles.about` instead. Otherwise, it is pretty hard to see that this is a translation key.

Reply:
Right now it's not a translations key, but used to create the translations key. But yeah, I can adjust so that the whole path is included here.